README.rst

```diff
@@ -257,6 +257,22 @@ Python code, and runs it using jail_code, modifying the globals dictionary as a
 side-effect.  safe_exec does this by serializing the globals into and out of
 the subprocess as JSON.
 
+Limitations
+-----------
+
+* If codejail or AppArmor is not configured properly, codejail will default to
+  running code insecurely (no sandboxing). It is not secure by default.
+* Resource limits can only be constrained using the mechanisms that Linux's
+  rlimit makes available. While rlimit can limit the size of any one file that
+  a process can create, and can limit the number of files it has open at any
+  one time, it cannot limit the total number of bytes written across *all* files.
+  A partial mitigation is to constrain the max execution time. (All files
+  written in the sandbox will be deleted at end of execution, in any case.)
+* Sandboxes do not have strong isolation from each other. Under proper
```

Comment:
What about something like "Sandbox isolation is achieved via AppArmor access limits and not inherint to codejail."

Reply:
As an addition, or a change to something that's there?

Reply:
It could be a replacement for the first sentence or maybe add a second sentence after the first that just says. "Sandbox isolation is achieved via AppArmor access limits and relies on AppArmor being setup correctly."